packages/viewer/src/commands/index.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @module Commands
+ */
+export * from './commandRegistry';
+
+export * from './streamCommands';
```

Comment:
added to define the module. without this, and including in the `typedoc.json`, the navigation ends up looking like the attached image. see comment on `src/index.ts` for more context.

![Screen Shot 2021-02-22 at 8 49 33 AM](https://user-images.githubusercontent.com/54546064/108736501-4cd98500-74f7-11eb-8e20-c86766003411.png)